v1/python-sdk/tutorials/automl-with-azureml/forecasting-hierarchical-timeseries/auto-ml-forecasting-hierarchical-timeseries.ipynb

```diff
@@ -251,8 +251,16 @@
    "source": [
     "### Set up training parameters\n",
     "\n",
-    "This dictionary defines the AutoML and hierarchy settings. For this forecasting task we need to define several settings inncluding the name of the time column, the maximum forecast horizon, the hierarchy definition, and the level of the hierarchy at which to train.\n",
+    "We need to provide ``ForecastingParameters`` and ``AutoMLConfig`` objects. For the forecasting task we need to define several settings including the name of the time column, the maximum forecast horizon, the hierarchy definition, and the level of the hierarchy at which to train.\n",
```

Comment:
Let's add the `ForecastingParameters` to the list of objects user needs to provide since hierarchy-related settings are set in the ForecastingParameters.

Reply:
Did you mean to say ``HTSTrainParameters``, since hierarchy-related settings are set in the HTSTrainParameters.
![image](https://user-images.githubusercontent.com/74648335/203275210-7f9b7a67-7fc3-435c-a4df-56099c13c6f6.png)

Reply:
@iamrk04 Yes, I meant that. Sorry for the confusion.